examples/README.md

```diff
@@ -4,6 +4,6 @@
 This directory contains examples of common use cases for MIGraphX. 
 
 ## Examples:
-- MIGraphX usage and utilities: [migraphx](./migraphx)
-- Vision inference examples: [vision](./vision)
-- Natural language inference examples: [nlp](./nlp)
\ No newline at end of file
+- MIGraphX usage and utilities: [MIGraphX usage and utilities](./migraphx)
+- Vision inference examples: [Vision inference examples](./vision)
+- Natural language inference examples: [Natural language inference examples](./nlp)
```

Comment:
You can just change them to (no need for the duplication):

- [MIGraphX usage and utilities](./migraphx)
- [Vision inference examples](./vision)
- [Natural language inference examples](./nlp)

Reply:
done